src/runtimes/node/bundlers/nft/es_modules.ts

```diff
@@ -49,6 +52,7 @@ const patchESMPackage = async (path: string, cache: RuntimeCache) => {
   return JSON.stringify(patchedPackageJson)
 }
 
+// eslint-disable-next-line complexity
```

Comment:
Can we move this to https://github.com/netlify/zip-it-and-ship-it/blob/main/.eslintrc.cjs?